### DISCUSSION TITLE
Any changes as a result of release 1884

### DISCUSSION THREAD

**Discussion by okadrian** [badstorm/surface-pro-7-opencore#82] [General]
I have a few little niggles on my SP7 using release 1882. 

After boot or opening the lid, the speakers will hiss until a sound is generated by the OS whereby the speakers click loudly then perform the intended audio signal.
The USB-C port reports having a theoretical max of 480Mbps.
The device battery drains quickly when supposedly in sleep mode. 

None of these are major issues but it'd be good to know if any of these are being experienced widely, and crucially, whether any have been resolved in this new release.

**Comment by Man1Design** [comment]
I experienced the same problems as you.
For audio, the problem does not occur if you have an app like spotify paused.

**Comment by Rainringgoaway** [comment]
1.Lould noises cause of surface H_EC Bigsurface (Battery status )applealc perhaps hackintosh MacOSsystem,When you active Bigsurface.kext without plugin charger (system recognize Battery)it came out noise
2.type-C 480Mb due to macbook usbmap default portspeed
